<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Training</class>
 <widget class="QDialog" name="Training">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber_2">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>159</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2p">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>159</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelp">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>159</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4p">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>159</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos4</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_7">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>279</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>279</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player8</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>159</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>279</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>159</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player5</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_3">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>159</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3p">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>159</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>310</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5p">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>344</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos5</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_5">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>344</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>102</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>102</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>129</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>129</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>129</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player2</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>102</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>102</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>8</y>
     <width>228</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Zig Zag / Pressing / Def Mirror</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>129</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player4</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_11">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>525</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11p">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>525</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos11</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_10">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>525</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10p">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>525</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9p">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>525</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>499</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player10</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_9">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>466</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>499</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player9</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_11">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>466</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_8">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>344</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8p">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>344</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos8</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>466</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>499</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player11</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12p">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>525</y>
     <width>37</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6p">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>344</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos6</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>466</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7p">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>344</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pos7</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>525</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_9">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>525</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_7">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>344</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_6">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>344</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>499</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player12</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>279</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="cost4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>493</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bugdet:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_16">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>493</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="cost1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy Left: </string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_13">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>525</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>9</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>557</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Theoretic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>250</y>
     <width>165</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Gym / Run / Barriers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>46</y>
     <width>147</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost: 5000$
Energy: -10
Goal: Defensive, Cardio </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>250</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>300OnRun / 350Static / 300Pivot </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>170</y>
     <width>91</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost: 8000$
Energy: -14
Goal: Shooting</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>290</y>
     <width>192</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost: 7000$
Energy: -18
Goal: Physical, Jump , Strength</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>410</y>
     <width>214</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost: 10000$
Energy: -5
Goal: Att-Def Analysis / Motivation </string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>20</width>
     <height>601</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>350</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>580</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: You can train your team just by clicking the desired training method. </string>
   </property>
  </widget>
  <widget class="QLabel" name="image_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>16</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>16</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>16</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>560</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="image_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>190</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
